regression/cbmc/array-cell-sensitivity15/test.desc

```diff
@@ -0,0 +1,10 @@
+CORE
+access.c
+--program-only
```

Comment:
I believe you need to add this test to the exclusion list in `regression/validate-trace-xml-schema/check.py` as `--program-only` isn't compatible with `--xml-ui`. That should make the regression tests pass.

Reply:
Done